app/src/main/java/org/oppia/android/app/devoptions/markchapterscompleted/MarkChaptersCompletedFragmentPresenter.kt

```diff
@@ -78,6 +78,17 @@ class MarkChaptersCompletedFragmentPresenter @Inject constructor(
               )
           }
         }
+      } else if (binding.isAllChecked == true) {
+        binding.isAllChecked = false
+        getMarkChaptersCompletedViewModel().getItemList().forEach { viewModel ->
+          if (viewModel is ChapterSummaryViewModel) {
+            if (!viewModel.checkIfChapterIsCompleted())
+              chapterUnselected(
+                viewModel.chapterIndex,
+                viewModel.nextStoryIndex
+              )
```

Comment:
Nit: I think this can fit within the 100 character limit, so it probably should be de-line wrapped so that it minimizes vertical space (for consistency with other code in the codebase).

Reply:
Hi @BenHenning when i de-line wrap i am getting lint error while committing the changes.

Reply:
Are you de-wrapping it like this:

```kotlin
              chapterUnselected(viewModel.chapterIndex, viewModel.nextStoryIndex)
```

That shouldn't result in a lint error as it fits within 100 characters, so I'm curious what issue you're seeing if that's what you're trying.

Reply:
This appears resolved in the latest changes.